core/domain/skill_services_test.py

```diff
@@ -126,6 +126,42 @@ def __init__(self, cmd: str, property_name: str) -> None:
             skill_services.apply_change_list(
                 self.SKILL_ID, invalid_skill_change_list, self.user_id_a)  # type: ignore[arg-type]
 
+    def test_apply_change_list_with_invalid_cmd_name_ignores_invalid_cmd_name(self) -> None:
```

Comment:
@nuflights-lkbhitesh07 
Do you have any suggestions of values we can pass in assert, since the test function only covers the branch (just ignores the cmd and continues). Additionally there is no catch of invalid cmd in the main skill_services class so that maybe I can assert if the error message returned is the exact message expected.

Reply:
@lkbhitesh07 I have implemented the changes. could you double check again?